This patch is required to fix file probe to support the has_extended_acl
attribute in OVAL.
This patch has not been contributed to upstream, but is planned to be
completed by 2015-Jan-31.
 
--- openscap-1.1.1/src/OVAL/probes/unix/file.c~1~	2014-10-27 12:41:33.138555312 -0700
+++ openscap-1.1.1/src/OVAL/probes/unix/file.c	2014-10-27 12:40:05.815261373 -0700
@@ -261,7 +261,7 @@
 	}
 	return (has_acl == 1) ? gr_true : gr_false;
 #elif defined(OS_SOLARIS)
-	return acl_trivial(st_path) ? gr_true : gr_false;
+	return acl_trivial(path) ? gr_true : gr_false;
 #else
 	return NULL;
 #endif
